### PR TITLE
Geojs

### PR DESCRIPTION
**Pull request by jeffbaumes**
Still WIP but can be tested out in a tabular dataset that has a lat, lon, and some sort of data field.

### CODE REVIEW

File: app/src/views/VisualizationManagementView.js [hunk 1]
```diff
@@ -98,17 +104,10 @@
...
         },
 
-        render: function () {
-            if (this.visualization) {
-                this.inputsView.collection.set(this.visualization.get('meta').visualization.inputs);
-                this.inputsView.render();
-            }
-            return this;
-        },
-
```
Comment (jeffbaumes) [inline app/src/views/VisualizationManagementView.js]: Looks like this needs to be added back in.

**Comment by zachmullen** [comment]
This feels like a good time to get geojs packaged in some subset of {bower, npm} :smile_cat:

**Comment by aashish24** [comment]
@zachmullen  its there already. try npm install geojs

**Comment by zachmullen** [comment]
@aashish24 Indeed that does work. We should depend on that in tangelohub's package.json instead of copying all the source into this repo.

**Comment by zachmullen** [comment]
And on a related note, we can serve the `girder.app.min.js` from the girder instance backing tangelohub rather than including its source in this repository.

(I'd like to get a girder front-end package into npm at some point but since it isn't much use without a girder server, I think fetching it from the static dir of girder is the best thing at this point.)

**Comment by jeffbaumes** [comment]
Zach I'm good with using hosted / npm libraries if you want to push some
changes here. There is a PR to geojs that is needed for multiple geojs view
resets to work - once that is in npm we can switch over.

**Comment by jeffbaumes** [comment]
This is waiting on GeoJS release with the initialize bug fix before taking out the GeoJS snapshot.

**Comment by jeffbaumes** [comment]
I think this is ready to go. @zackgalbreath could you test with visualizing a simple csv file like:

```
lat,lon,size,color
0,0,1,a
10,10,1,b
0,10,5,b
10,0,10,a
```

**Comment by curtislisle** [comment]
Cool...

**Comment by zackgalbreath** [comment]
Manual test confirmed, LGTM

**Comment by aashish24** [comment]
Nice!.. I will run it manually as well..

**Pull request merged**